waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -63,11 +64,11 @@ genPrismaSchema ::
   Generator FileDraft
 genPrismaSchema spec = do
   (datasourceProvider :: String) <- case dbSystem of
-    AS.Db.PostgreSQL -> return "postgresql"
+    AS.Db.PostgreSQL -> return (Pls.Db.dbToPrismaProviderId AS.Db.PostgreSQL)
```

Comment:
Ha yes, even here this is too much. You already cased on PostgreSQL -> you know what you need now here is prisma's PSL keyword for postgresql. So you could just do something like `Wasp.Prisma.PSL.postgresqlIdentifier` or Keyword or whatever is better word and that is it. Same below for SQlite.

If you were instead in a situaiton where you are not doing the casing here, and you want to do `Pls.Db.dbToPrismaProviderId dbSystem`, then it would make sense to have `Pls.Db.dbToPrismaProviderId` (although I would name it better), but we don't actually need taht anywehere from what I see.